src/Integration.UnitTests/Service/SonarQubeServiceWrapperTests.cs

```diff
@@ -464,6 +464,39 @@ public void SonarQubeServiceWrapper_GetPlugins()
             }
         }
 
+        [TestMethod]
+        public void SonarQubeServiceWrapper_CreateProjectDashboardUrl()
+        {
+            // Setup
+            var testSubject = new SonarQubeServiceWrapper(this.serviceProvider);
+
+            var serverUrl = new Uri("http://my-sonar-server:5555");
+            var connectionInfo = new ConnectionInformation(serverUrl);
+            var projectInfo = new ProjectInformation { Key = "p1" };
+
+            Uri expectedUrl = new Uri(serverUrl, string.Format(SonarQubeServiceWrapper.ProjectDashboardRelativeUrl, projectInfo.Key));
```

Comment:
@duncanpMS: I've now used a hard string as suggested. Should be noted that we do this sort of thing in quite a few of the other tests, mainly related to API calls.